debian/rules

```diff
@@ -3,6 +3,17 @@
 %:
 	dh $@
 
+.PHONY: override_dh_strip
+override_dh_strip:
+	dh_strip -plibkano-profile --dbg-package=libkano-profile-dbg
+	dh_strip -plibkano-world --dbg-package=libkano-world-dbg
+
 override_dh_auto_build:
 	bash media/images/clean-images.sh
 	cd lxpanel-plugin && make
+
+	cd libs/kano-profile && make
+	cd libs/kano-profile && make debug
+
+	cd libs/kano-world && make
+	cd libs/kano-world && make debug
```

Comment:
@skarbat The Makefile should compile everything into the `debug` and `release` subdirectories so the module objects shouldn't be present to require the rebuild. I tried building the package with `dr` and it seemed fine but if we notice some problems with the debug symbols missing then I'll add it in.